frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/review-page.tsx

```diff
@@ -54,7 +59,10 @@ const ReviewPage: React.FC<ReviewPageProps> = ({ state }) => {
           </ReviewListBody>
           <ReviewListBody hideIcon>
             <span>{t('ceph-storage-plugin~Time to live: ')}</span>&nbsp;
-            <span className="text-secondary">{`${timeToLive} ms`}</span>
+            <span className="text-secondary">{`${convertMsToUnit(
```

Comment:
Have you tried pluralising it through i18n ? check how its being done in multi-namespace-store.tsx L84 or try using `import { pluralize } from '@patternfly/react-core';` instead of creating a new function